### DISCUSSION TITLE
How to edit a value in multiple rows?

### DISCUSSION THREAD

**Discussion by thomaskyhn** [sqlitebrowser/sqlitebrowser#2771] [Q&A] [answered]
Is it possible to edit multiple cells simultaneously? I need to change a vale that occurs more than 2000 times in the same column, but so far I've only been able to do it one cell at a time.

**Comment by chrisjlocke** [comment, marked as answer]
Depends on what you're editing, and what common reference all the rows have.
Usually you can edit fields A, B and C for every row where D equals X, E is less than Y and also F is more than Z.  Maybe you have do these repeatedly.
You can accomplish this via SQL statements.
You can't edit multiple cells at the same time using the GUI, but you can paste a value from the clipboard into multiple cells.  Select one cell, copy the value (press CTRL + C if on Windows), select a range of cells (either horizontally, vertically or even both) and press CTRL + V.

**Reply by thomaskyhn** [reply]
Thanks so much! I see now that pasting a new value works.
